app/templates/view_buyers.html

```diff
@@ -0,0 +1,85 @@
+{% import "toolkit/summary-table.html" as summary %}
+
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+    Buyers – Digital Marketplace admin
+{% endblock %}
+
+{% block breadcrumb %}
+  {%
+      with items = [
+          {
+              "link": url_for('.index'),
+              "label": "Admin home"
+          },
+      ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  {% with messages = get_flashed_messages(with_categories=true) %}
+  {% if messages %}
+      {% for category, message in messages %}
+          {% if message == 'no_opportunity' %}
+              {% set displayed_message = "Sorry, we couldn't find an opportunity with the ID: {}".format(opportunity_id) %}
```

Comment:
We shouldn't be apologising, better to be more precise  'There are no briefs with ID'